vendor/github.com/portworx/kdmp/pkg/controllers/dataexport/reconcile.go

```diff
@@ -267,7 +267,7 @@ func (c *Controller) sync(ctx context.Context, in *kdmpapi.DataExport) (bool, er
 		var compressionType string
 		var podDataPath string
 		var excludeFileList string
-		pvcStorageClass := dataExport.Annotations[kdmpStorageClassKey]
+		pvcStorageClass := dataExport.Labels[kdmpStorageClassKey]
```

Comment:
This seem to be an unrelated change came because @vikasit12 has merged an chnage directly to the vendor file of stork. It should come via kdmp vendoring to stork. 
It seems @pallav-px  is already raising a PR to correct this up ,so that vendoring will change this file.